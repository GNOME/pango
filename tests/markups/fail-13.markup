<span show="all">test</span>
