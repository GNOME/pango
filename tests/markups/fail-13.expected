ERROR: 'all' is not a valid value for the 'show' attribute on <span> tag, line 1; valid values are none/spaces/line-breaks/ignorables or combinations with |